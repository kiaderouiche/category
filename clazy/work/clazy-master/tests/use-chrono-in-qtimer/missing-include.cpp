#include <QtCore/QObject>
#include <QtCore/QString>
#include <QtCore/QTimer>

void test()
{
    QTimer timer;
    timer.setInterval(1000); // Warn
}
