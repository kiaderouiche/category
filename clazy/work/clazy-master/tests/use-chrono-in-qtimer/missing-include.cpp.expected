use-chrono-in-qtimer/missing-include.cpp:8:23: warning: make code more robust: use 1s instead. [-Wclazy-use-chrono-in-qtimer]
